### DISCUSSION TITLE
URGENT: Github Account Suspended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#136382] [Other Feature Feedback, Questions, & Ideas]
### Github Account Suspended

General

### Body

Today when i was working on a hackathon project .Suddenly i got suspended.I didnt get any reason why my account got suspended from github.
As a college student i use github for fair use, i do projects and upload them in github.
I have mailed the support 
Ticket id :  2951873
Github username : Rudra-Sankha-Sinhamahapatra

Can anybody assist me how to proceed further to recover my github account

Thanks

**Comment by GabrielAlmeidag** [comment]
@rushhaaland 😺👋

I'm sorry to hear about the unexpected suspension of your GitHub account. It seems you've already contacted support, which is a great step. Check your email, including your spam folder, to see if GitHub has sent any explanation or additional requests. Also, make sure to review GitHub's Terms of Service to ensure that your activities and projects are compliant. When responding to support, provide clear details about your use of the account for academic and hackathon projects. Be patient, as support responses can take some time. For further assistance, you can contact GitHub Support at [support@github.com](mailto:support@github.com).

**Reply by ghost** [reply]
I didnt get any mail from github on either my inbox or spam box
And i have seen my projects they were legit and simple 
I am waiting for github team's reply because my github account have lot of projects ,and i am doing hardwork and actively using github since past 1 year.
Hope they soon reply me through mail

**Comment by LiteBrite82** [comment]
Hi @rushhaaland 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 


Thank you!
